platforms/documentation/docs/src/docs/userguide/optimizing-builds/configuration-cache/configuration_cache_enabling.adoc

```diff
@@ -68,7 +68,7 @@ org.gradle.configuration-cache.problems=warn
 
 [WARNING]
 ====
-The warning mode is a migration and troubleshooting aid and not intended as a persistent way of ignoring incompatibilities.
+Warning mode is a migration and troubleshooting aid and not intended as a persistent way of ignoring incompatibilities.
```

Comment:
This section is also duplicated here: https://docs.gradle.org/nightly/userguide/configuration_cache_debugging.html#enable_warning_mode

Reply:
> is it a collection of topics related to fighting a particular problem?

It is an "all things CC" debug page. At least that's what I was going for.